### PR TITLE
OracleDB 12c Integration

### PR DESCRIPTION
**Pull request by NicoGuini**
|Related issue|
|---|
|https://github.com/wazuh/wazuh/issues/9056|

## Description

OracleDB 12C integration.

## UT
No UT can be created. The limitation is logtest doesn't allows multiline logs

Tested with Wazuh 4.2

### CODE REVIEW

**Review by jcruzlp — APPROVED** [review]

File: ruleset/rules/0920-oracledb_rules.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    -->
+    <rule id="89101" level="3">
```
Comment (fabamatic) [inline ruleset/rules/0920-oracledb_rules.xml:21]: As a suggestion I would merge rules 81900 and 81901 unless another reason for rule granularity shows up

Comment (NicoGuini) [reply ruleset/rules/0920-oracledb_rules.xml]: Done

**Review by JavierBejMen — APPROVED** [review]
LGTM

**Review by fabamatic — APPROVED** [review]
Looking good

**Review by 72nomada — APPROVED** [review]
lgtm

**Pull request merged**